### ISSUE TITLE
Would the 5b_lyrics model run on a RTX 3090?

### ISSUE THREAD

**Issue by Flesco** [openai/jukebox#150]
RTX 3090 has 24gb memory. This would be enough to run the 5b model, correct?

**Comment by johndpope** [comment]
I don't think it will - not even 2 cards will help. Seems like it's in the realm of rtx 8000 exclusively. happy to be corrected here. https://github.com/openai/jukebox/issues/142

**Comment by AeroScripts** [comment]
It runs (inference) on my 3090 without issue, haven't tried training though.

keep in mind pytorch hasn't been updated to the new cuda yet so you have to compile it yourself

**Comment by Flesco** [comment]
> It runs (inference) on my 3090 without issue, haven't tried training though.
> 
> keep in mind pytorch hasn't been updated to the new cuda yet so you have to compile it yourself

Good to know, thanks for sharing! And please let me know if training works, if you attempt it.

**Comment by AeroScripts** [comment]
Also worth mentioning my system memory (32gb) can just barely handle it, during init, it fills up fully and has to swap about 2gb, but once its loaded that goes down to about 20gb usage

**Comment by Flesco** [comment]
One more question @AeroScripts - how fast is it? For example, how long does it take to generate 20 seconds of music?

**Comment by Randy-H0** [comment]
> One more question @AeroScripts - how fast is it? For example, how long does it take to generate 20 seconds of music?

I can say that on the google colab version it takes about a minute for a second not upsampled.
It takes 3-5 hours for 60 seconds for 1 file upsampled.

The RTX 3090 is still really good but it will really not go that fast as the GPU's google are using.

**Comment by moih** [comment]
@Randy1435 @AeroScripts Have you managed to train a Small Prior using the RTX 3090?

I have been trying but get a cuDNN error (NOT an OOM); I'd really appreciate it if you could pass on a fix for it? Thanks!

**Comment by lucaguarro** [comment]
Any update on this?
